### PR TITLE
ASimProcessCreateMicrosoftSysmon.yaml-48

### PR DESCRIPTION
**Pull request by Alekhya0824**
Required items, please complete
   
   Change(s):
   - Created a new file for WindowsEvent and added WindowsEvent parser in it. In Event file removed WindowsEvent parser and in ASimProcessEvent added WindowsEvent Parser name  and same changes have done for vim as well.

   Reason for Change(s):
   - To Split the WindowsEvent and Event parser separately

   Version Updated:
   - yes

   Testing Completed:
   - yes

   Checked that the validations are passing and have addressed any issues that are present:
   - yes

### CODE REVIEW

**Comment by v-atulyadav** [comment]
Hi @Alekhya0824,
Please check for failure validation. Thanks
image

**Comment by v-atulyadav** [comment]
Hi @Alekhya0824,
Please update on above. Thanks

**Comment by v-atulyadav** [comment]
Hi @Alekhya0824,
Validations failing please check. Thanks

File: Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmon.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Parser:
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmon.yaml:1]: why we are changing KQL tests

File: Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmon.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
   Title: Process Create Event ASIM parser for Sysmon
-  Version: '0.3.2'
-  LastUpdated: Feb 23, 2022
+  Version: '0.4.1'
+  LastUpdated: June 3, 2024
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmon.yaml:4]: "LastUpdated" property exists but is not correct format. The expected format is, for example, "Jun 29, 2024"

File: Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmon.yaml [hunk 3]
```diff
@@ -13,7 +13,7 @@ References:
 - Title: ASIM
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmon.yaml:13]: ASIM schema "ProcessEvent" version "0.1.0" is incorrect. The correct version for ASIM schema "ProcessEvent" is "0.1.4"

File: Parsers/ASimProcessEvent/Parsers/vimProcessCreateMicrosoftSysmon.yaml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 Parser:
   Title: Process Create Event ASIM parser for Sysmon
-  Version: '0.4.0'
-  LastUpdated: Feb 23, 2022
+  Version: '0.5.1'
+  LastUpdated: June 3, 2024
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/vimProcessCreateMicrosoftSysmon.yaml:4]: "LastUpdated" property exists but is not correct format. The expected format is, for example, "Jun 29, 2024"

File: Parsers/ASimProcessEvent/Parsers/vimProcessCreateMicrosoftSysmon.yaml [hunk 5]
```diff
@@ -13,7 +13,7 @@ References:
 - Title: ASIM
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/vimProcessCreateMicrosoftSysmon.yaml:13]: ASIM schema "ProcessEvent" version "0.1.0" is incorrect. The correct version for ASIM schema "ProcessEvent" is "0.1.4"

File: Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmon.yaml [hunk 6]
```diff
@@ -13,7 +13,7 @@ References:
...
   Link: https://aka.ms/AboutASIM
 Description: |
-  This ASIM parser supports normalizing Sysmon process create events (event 1) collected using the Event or WEF (WindowsEvent table) connectors to the ASIM Process Event normalized schema. 
+  This ASIM parser supports normalizing Sysmon process create events (event 1) collected using the Event or WEF (WindowsEvent table) connectors logs ingested in 'Event' table to the ASIM Process Event normalized schema.
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmon.yaml:16]: please review description

File: Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmonWindowsEvent.yaml [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+  Schema: ProcessEvent
+  Version: '0.1.0'
+References:
+- Title: ASIM Process Schema
+  Link: https://aka.ms/ASimProcessEventDoc
+- Title: ASIM
+  Link: https://aka.ms/AboutASIM
+Description: |
+  This ASIM parser supports normalizing Sysmon process create events (event 1) collected using the Event or WEF (WindowsEvent table) connectors logs ingested in 'WindowsEvent' table to the ASIM Process Event normalized schema. 
+ParserName: ASimProcessCreateMicrosoftSysmonWindowsEvent
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmonWindowsEvent.yaml:17]: ASimProcessEventCreateMicrosoftSysmonWindowsEvent

File: Parsers/ASimProcessEvent/Parsers/vimProcessCreateMicrosoftSysmonWidosEvent.yaml [hunk 8]
```diff
@@ -0,0 +1,192 @@
...
+  Version: '0.1.0'
+References:
+- Title: ASIM Process Schema
+  Link: https://aka.ms/ASimProcessEventDoc
+- Title: ASIM
+  Link: https://aka.ms/AboutASIM
+Description: |
+  This ASIM parser supports normalizing Sysmon process create events (event 1) collected using the Event or WEF (WindowsEvent table) connectors logs ingested in 'WindowsEvent' table to the ASIM Process Event normalized schema. 
+ParserName: vimProcessCreateMicrosoftSysmonWindowsEvent
+EquivalentBuiltInParser: _Im_ProcessCreate_MicrosoftSysmonWindowsEvent
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/vimProcessCreateMicrosoftSysmonWidosEvent.yaml:18]: _Im_ProcessEvent_CreateMicrosoftSysmonWindowsEvent

File: Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmon.yaml [hunk 9]
```diff
@@ -13,7 +13,7 @@ References:
 - Title: ASIM
   Link: https://aka.ms/AboutASIM
 Description: |
-  This ASIM parser supports normalizing Sysmon process create events (event 1) collected using the Event or WEF (WindowsEvent table) connectors to the ASIM Process Event normalized schema. 
+  This ASIM parser supports normalizing Sysmon process create events (event 1) collected using the Event or WEF (WindowsEvent table) connectors logs ingested in 'Event' table to the ASIM Process Event normalized schema. 
 ParserName: ASimProcessCreateMicrosoftSysmon
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmon.yaml:17]: ASimProcessEventCreateMicrosoftSysmon

File: Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmonWindowsEvent.yaml [hunk 10]
```diff
@@ -0,0 +1,91 @@
+Parser:
+  Title: Process Create Event ASIM parser for Sysmon
+  Version: '0.4.1'
+  LastUpdated: June 3, 2024
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmonWindowsEvent.yaml:4]: date

File: Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmonWindowsEvent.yaml [hunk 11]
```diff
@@ -0,0 +1,91 @@
...
+Product:
+  Name: Sysmon
+Normalization:
+  Schema: ProcessEvent
+  Version: '0.1.0'
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmonWindowsEvent.yaml:9]: ASIM schema "ProcessEvent" version "0.1.0" is incorrect. The correct version for ASIM schema "ProcessEvent" is "0.1.4"

File: Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmonWindowsEvent.yaml [hunk 12]
```diff
@@ -0,0 +1,91 @@
...
+References:
+- Title: ASIM Process Schema
+  Link: https://aka.ms/ASimProcessEventDoc
+- Title: ASIM
+  Link: https://aka.ms/AboutASIM
+Description: |
+  This ASIM parser supports normalizing Sysmon process create events (event 1) collected using the Event or WEF (WindowsEvent table) connectors logs ingested in 'WindowsEvent' table to the ASIM Process Event normalized schema.
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmonWindowsEvent.yaml:16]: please review description

File: Parsers/ASimProcessEvent/Parsers/vimProcessCreateMicrosoftSysmon.yaml [hunk 13]
```diff
@@ -13,7 +13,7 @@ References:
...
   Link: https://aka.ms/AboutASIM
 Description: |
-  This ASIM parser supports filtering and normalizing Sysmon process create events (event 1) collected using the Event or WEF (WindowsEvent table) connectors to the ASIM Process Event normalized schema. 
+  This ASIM parser supports normalizing Sysmon process create events (event 1) collected using the Event or WEF (WindowsEvent table) connectors logs ingested in 'Event' table to the ASIM Process Event normalized schema.
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/vimProcessCreateMicrosoftSysmon.yaml:16]: please review description

File: Parsers/ASimProcessEvent/Parsers/vimProcessCreateMicrosoftSysmonWidosEvent.yaml [hunk 14]
```diff
@@ -0,0 +1,192 @@
...
+Normalization:
+  Schema: ProcessEvent
+  Version: '0.1.0'
+References:
+- Title: ASIM Process Schema
+  Link: https://aka.ms/ASimProcessEventDoc
+- Title: ASIM
+  Link: https://aka.ms/AboutASIM
+Description: |
+  This ASIM parser supports normalizing Sysmon process create events (event 1) collected using the Event or WEF (WindowsEvent table) connectors logs ingested in 'WindowsEvent' table to the ASIM Process Event normalized schema.
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/vimProcessCreateMicrosoftSysmonWidosEvent.yaml:16]: please review description

File: Parsers/ASimProcessEvent/test/Create_MicrosoftSysmon_ASimProcessCreate_DataTest.csv [hunk 15]
```diff
@@ -0,0 +1,2 @@
+﻿Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 10 records (100.0%) for field [HashType] of type [Enumerated]: [""SHA256""] (Schema:ProcessEvent)"
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/test/Create_MicrosoftSysmon_ASimProcessCreate_DataTest.csv:2]: please check this error

File: Parsers/ASimProcessEvent/test/Create_MicrosoftSysmon_ASimProcessCreate_SchemaTest.csv [hunk 16]
```diff
@@ -0,0 +1,102 @@
+﻿Result
+"(1) Warning: Missing recommended field [ActorUserId]"
+"(1) Warning: Missing recommended field [DvcDomain]"
+"(1) Warning: Missing recommended field [DvcIpAddr]"
+"(1) Warning: Missing recommended field [EventUid]"
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/test/Create_MicrosoftSysmon_ASimProcessCreate_SchemaTest.csv:5]: please map this

File: Parsers/ASimProcessEvent/test/Create_MicrosoftSysmon_VimProcessCreate_DataTest.csv [hunk 17]
```diff
@@ -0,0 +1,2 @@
+﻿Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 10 records (100.0%) for field [HashType] of type [Enumerated]: [""SHA256""] (Schema:ProcessEvent)"
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/test/Create_MicrosoftSysmon_VimProcessCreate_DataTest.csv:2]: please check error

File: Parsers/ASimProcessEvent/test/Create_MicrosoftSysmon_VimProcessCreate_SchemaTest.csv [hunk 18]
```diff
@@ -0,0 +1,102 @@
+﻿Result
+"(1) Warning: Missing recommended field [ActorUserId]"
+"(1) Warning: Missing recommended field [DvcDomain]"
+"(1) Warning: Missing recommended field [DvcIpAddr]"
+"(1) Warning: Missing recommended field [EventUid]"
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/test/Create_MicrosoftSysmon_VimProcessCreate_SchemaTest.csv:5]: please map this field

File: Parsers/ASimProcessEvent/test/Create_MicrosoftSysmon_Windows_Event_ASimProcessCreate_SchemaTest.csv [hunk 19]
```diff
@@ -0,0 +1,101 @@
+﻿Result
+"(1) Warning: Missing recommended field [ActorUserId]"
+"(1) Warning: Missing recommended field [DvcDomain]"
+"(1) Warning: Missing recommended field [DvcIpAddr]"
+"(1) Warning: Missing recommended field [EventUid]"
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/test/Create_MicrosoftSysmon_Windows_Event_ASimProcessCreate_SchemaTest.csv:5]: please map this field

File: Parsers/ASimProcessEvent/test/Create_MicrosoftSysmon_Windows_Event_VimProcessCreate_SchemaTest.csv [hunk 20]
```diff
@@ -0,0 +1,101 @@
+﻿Result
+"(1) Warning: Missing recommended field [ActorUserId]"
+"(1) Warning: Missing recommended field [DvcDomain]"
+"(1) Warning: Missing recommended field [DvcIpAddr]"
+"(1) Warning: Missing recommended field [EventUid]"
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/test/Create_MicrosoftSysmon_Windows_Event_VimProcessCreate_SchemaTest.csv:5]: please map this field

**Review by vakohl — CHANGES_REQUESTED** [review]

File: Parsers/ASimProcessEvent/Parsers/ASimProcessEvent.yaml [hunk 21]
```diff
@@ -1,7 +1,7 @@
 Parser:
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/ASimProcessEvent.yaml:1]: we would need to update this file as well:
https://github.com/Azure/Azure-Sentinel/blob/23b502e285b220f319d6ebea53118a10e8025c79/Parsers/ASimProcessEvent/Parsers/ASimProcessEventCreate.yaml

File: Parsers/ASimProcessEvent/Parsers/imProcessEvent.yaml [hunk 22]
```diff
@@ -1,7 +1,7 @@
 Parser:
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/imProcessEvent.yaml:1]: please update this file as well:
https://github.com/Azure/Azure-Sentinel/blob/23b502e285b220f319d6ebea53118a10e8025c79/Parsers/ASimProcessEvent/Parsers/imProcessCreate.yaml

**Review by vakohl — CHANGES_REQUESTED** [review]

**Comment by v-atulyadav** [comment]
Hi @Alekhya0824,
Validations failures in this request, please check with them. And also, one suggestion: once you're done with the open comment change, please click on the `Resolve Conversation` button.

File: Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmon.yaml (hunk 1 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmon.yaml:1]: updating table schema file

File: Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmon.yaml (hunk 2 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmon.yaml]: done

File: Parsers/ASimProcessEvent/Parsers/vimProcessCreateMicrosoftSysmon.yaml (hunk 4 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/vimProcessCreateMicrosoftSysmon.yaml]: done

**Comment by v-atulyadav** [comment]
Hi @Alekhya0824,
Please work on open comments and also check for failed validations. Thanks

File: Parsers/ASimProcessEvent/Parsers/vimProcessCreateMicrosoftSysmon.yaml (hunk 5 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/vimProcessCreateMicrosoftSysmon.yaml:13]: DONE

File: Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmonWindowsEvent.yaml (hunk 7 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmonWindowsEvent.yaml]: DONE

File: Parsers/ASimProcessEvent/Parsers/vimProcessCreateMicrosoftSysmonWidosEvent.yaml (hunk 8 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/vimProcessCreateMicrosoftSysmonWidosEvent.yaml]: DONE

File: Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmonWindowsEvent.yaml (hunk 10 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmonWindowsEvent.yaml]: DONE

File: Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmonWindowsEvent.yaml (hunk 12 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmonWindowsEvent.yaml]: DONE

File: Parsers/ASimProcessEvent/Parsers/vimProcessCreateMicrosoftSysmon.yaml (hunk 13 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/vimProcessCreateMicrosoftSysmon.yaml]: DONE

File: Parsers/ASimProcessEvent/Parsers/vimProcessCreateMicrosoftSysmonWidosEvent.yaml (hunk 14 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/vimProcessCreateMicrosoftSysmonWidosEvent.yaml]: DONE

File: Parsers/ASimProcessEvent/Parsers/ASimProcessEvent.yaml (hunk 21 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/ASimProcessEvent.yaml:1]: DONE

File: Parsers/ASimProcessEvent/Parsers/imProcessEvent.yaml (hunk 22 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/imProcessEvent.yaml:1]: DONE

File: Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmonWindowsEvent.yaml (hunk 11 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmonWindowsEvent.yaml]: DONE

File: Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmon.yaml (hunk 9 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmon.yaml]: DONE

File: Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmon.yaml (hunk 6 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmon.yaml]: DONE

File: Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmon.yaml (hunk 3 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/ASimProcessCreateMicrosoftSysmon.yaml:13]: DONE

File: Parsers/ASimProcessEvent/test/Create_MicrosoftSysmon_VimProcessCreate_DataTest.csv (hunk 17 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/test/Create_MicrosoftSysmon_VimProcessCreate_DataTest.csv:2]: updated supported values in tester file

File: Parsers/ASimProcessEvent/test/Create_MicrosoftSysmon_ASimProcessCreate_SchemaTest.csv (hunk 16 above)
Comment (vakohl) [reply Parsers/ASimProcessEvent/test/Create_MicrosoftSysmon_ASimProcessCreate_SchemaTest.csv:5]: EventUid

File: Parsers/ASimProcessEvent/test/Create_MicrosoftSysmon_VimProcessCreate_SchemaTest.csv (hunk 18 above)
Comment (vakohl) [reply Parsers/ASimProcessEvent/test/Create_MicrosoftSysmon_VimProcessCreate_SchemaTest.csv:5]: EventUid

File: Parsers/ASimProcessEvent/test/Create_MicrosoftSysmon_Windows_Event_ASimProcessCreate_SchemaTest.csv (hunk 19 above)
Comment (vakohl) [reply Parsers/ASimProcessEvent/test/Create_MicrosoftSysmon_Windows_Event_ASimProcessCreate_SchemaTest.csv:5]: EventUid

File: Parsers/ASimProcessEvent/test/Create_MicrosoftSysmon_Windows_Event_VimProcessCreate_SchemaTest.csv (hunk 20 above)
Comment (vakohl) [reply Parsers/ASimProcessEvent/test/Create_MicrosoftSysmon_Windows_Event_VimProcessCreate_SchemaTest.csv:5]: EventUid

File: Parsers/ASimProcessEvent/test/Create_MicrosoftSysmon_ASimProcessCreate_SchemaTest.csv (hunk 16 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/test/Create_MicrosoftSysmon_ASimProcessCreate_SchemaTest.csv]: done

File: Parsers/ASimProcessEvent/test/Create_MicrosoftSysmon_Windows_Event_VimProcessCreate_SchemaTest.csv (hunk 20 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/test/Create_MicrosoftSysmon_Windows_Event_VimProcessCreate_SchemaTest.csv]: done

File: Parsers/ASimProcessEvent/test/Create_MicrosoftSysmon_Windows_Event_ASimProcessCreate_SchemaTest.csv (hunk 19 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/test/Create_MicrosoftSysmon_Windows_Event_ASimProcessCreate_SchemaTest.csv]: done

File: Parsers/ASimProcessEvent/test/Create_MicrosoftSysmon_VimProcessCreate_SchemaTest.csv (hunk 18 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/test/Create_MicrosoftSysmon_VimProcessCreate_SchemaTest.csv]: done

File: Parsers/ASimProcessEvent/Parsers/vimProcessCreateMicrosoftSysmon.yaml [hunk 23]
```diff
@@ -1,21 +1,21 @@
...
   Link: https://aka.ms/ASimProcessEventDoc
 - Title: ASIM
   Link: https://aka.ms/AboutASIM
 Description: |
-  This ASIM parser supports filtering and normalizing Sysmon process create events (event 1) collected using the Event or WEF (WindowsEvent table) connectors to the ASIM Process Event normalized schema. 
-ParserName: vimProcessCreateMicrosoftSysmon
-EquivalentBuiltInParser: _Im_ProcessCreate_MicrosoftSysmon
+  This ASIM parser supports normalizing Sysmon process create events (event 1) collected logs ingested in 'Event' table to the ASIM Process Event normalized schema. 
+ParserName: vimProcessEventCreateMicrosoftSysmon
+EquivalentBuiltInParser: _Im_ProcessEvent_Create_MicrosoftSysmon
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/vimProcessCreateMicrosoftSysmon.yaml:18]: _Im_ProcessEvent_CreateMicrosoftSysmon

File: Parsers/ASimProcessEvent/Parsers/vimProcessCreateMicrosoftSysmonWidowsEvent.yaml [hunk 24]
```diff
@@ -0,0 +1,193 @@
...
+  Version: '0.1.0'
+References:
+- Title: ASIM Process Schema
+  Link: https://aka.ms/ASimProcessEventDoc
+- Title: ASIM
+  Link: https://aka.ms/AboutASIM
+Description: |
+  This ASIM parser supports normalizing Sysmon process create events (event 1) collected using the Event or WEF (WindowsEvent table) connectors logs ingested in 'WindowsEvent' table to the ASIM Process Event normalized schema. 
+ParserName: vimProcessEventCreateMicrosoftSysmonWindowsEvent
+EquivalentBuiltInParser: _Im_ProcessEvent_Create_MicrosoftSysmonWindowsEvent
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/vimProcessCreateMicrosoftSysmonWidowsEvent.yaml:18]: _Im_ProcessEvent_CreateMicrosoftSysmonWindowsEvent

File: Parsers/ASimProcessEvent/Parsers/ASimProcessEvent.yaml (hunk 21 above)
Comment (vakohl) [reply Parsers/ASimProcessEvent/Parsers/ASimProcessEvent.yaml:1]: not visible. Please validate

Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/ASimProcessEvent.yaml:1]: added

File: Parsers/ASimProcessEvent/Parsers/imProcessEvent.yaml (hunk 22 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/imProcessEvent.yaml:1]: added

File: Parsers/ASimProcessEvent/Parsers/vimProcessCreateMicrosoftSysmonWidowsEvent.yaml (hunk 24 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/vimProcessCreateMicrosoftSysmonWidowsEvent.yaml]: DONE

File: Parsers/ASimProcessEvent/Parsers/vimProcessCreateMicrosoftSysmon.yaml (hunk 23 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/vimProcessCreateMicrosoftSysmon.yaml]: DONE

**Review by vakohl — APPROVED** [review]

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**